src/App.jsx

```diff
@@ -2,18 +2,47 @@ import { useState } from 'react'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
   const [name, setName] = useState("");
-  const [day, setDay] = useState(0)
+  const [day, setDay] = useState(1)
   const [month, setMonth] = useState('')
+  const [daysToGo, setDaysToGo] = useState(0)
+  const [isNotSubmitted, setIsNotSubmitted] = useState(true)
 
+  const dayMin = 1
+  const dayMax = 31
 
+  const today = new Date()
+  let birthday;
+
+  const updateDay = event => {
+    const birthdayDay = Math.max(dayMin, Math.min(dayMax, Number(event.target.value)))
+    setDay(birthdayDay)
+  }
+
+  const submit = () => {
+    setIsNotSubmitted(false)
+    birthday = new Date(`${month.toString()}-${day.toString()}`)
+    let daysDifference = Math.ceil((birthday.getTime() - today.getTime()) / (1000 * 3600 * 24)).toFixed(0)
+    console.log(daysDifference)
+    while (daysDifference < -367) {
```

Comment:
What's with -367? Seems like a strange number? 

Also how does the diff in days be more than 367 days different?

Reply:
Is this something to do with this year being a leap year? What happens in a non-leap year